strr-web/pages/create-account.vue

```diff
@@ -186,6 +189,11 @@ const setActiveStep = (newStep: number) => {
   activeStep.value.step.complete = true
   activeStepIndex.value = newStep
   activeStep.value = steps[activeStepIndex.value]
+  // TODO: remove
```

Comment:
Left over code, or just making step 1 valid? Then TODO comment should be different.

Reply:
Once the step 1 [PM] is implemented, I will add the validations for that. For now, this is making that step always valid and complete so that applications can be still be submitted.

Reply:
The same applies to the TODO comment in `steps.ts`.

Reply:
👍🏻